src/tests/fuzz/fuzz_sigpcap.c

```diff
@@ -163,6 +163,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
     p->ts.tv_sec = header->ts.tv_sec;
     p->ts.tv_usec = header->ts.tv_usec % 1000000;
     p->datalink = pcap_datalink(pkts);
+    p->pkt_src = PKT_SRC_WIRE;
```

Comment:
Should this be done for fuzz_predefpcap_aware.c as well ? 
Looks done already for fuzz_sigpcap_aware.c

Reply:
Yeah probably. If I run anything with `--enable-debug` it reaches an assertion otherwise.

Reply:
Ok, I never ran fuzzing with debug (only debug validation)

Reply:
It's just when I'm running the reproducer, then I have debug on.